charts/sck-otel/templates/_config.tpl

```diff
@@ -84,6 +84,14 @@ extensions:
 #   The simplest way to specify the ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable.
     size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
 receivers:
+
+# https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver
+  {{- if .Values.journaldLogs.enabled }}
+  journald:
+    directory: {{- toYaml .Values.journaldLogs.directory | nindent 6 }}
+    units: {{- toYaml .Values.journaldLogs.units | nindent 8 }}
+    priority: {{- toYaml .Values.journaldLogs.priority | nindent 6 }}
```

Comment:
why do you need `nindent` ? 
`units` is an arrayy. does it get handled well ?

Reply:
for indentation and new line to get the below format? 
```
receivers:
  journald:
    directory: /run/log/journal
    priority: info
    units:
    - kubelet
    - docker
    - containerd
```
Should this be handled using diff helm functions?